tensorrt_llm/llmapi/llm_args.py

```diff
@@ -2593,6 +2593,12 @@ class TorchLlmArgs(BaseLlmArgs):
         status="prototype",
     )
 
+    worker_extension_cls: Optional[str] = Field(
```

Comment:
How about renaming it to `ray_worker_extension_cls`? It seems a Ray-specific feature, and by making the name explicit, it is clearer to use and refactor later.

Reply:
Changed.